js/data.js

```diff
@@ -1,9 +1,16 @@
-import { getRandomIntegerRange, getRandomArrayElement } from './convert';
+import { getRandomIntegerRange, getRandomArrayElement } from './convert.js';
 // Данные для объекта
 const OFFER_TITLE = [
   'Большая квартира',
+  'Теплое шале',
+  'Уютная студия',
+  'С высоты птичего полета',
 ];
-const avatarNumbers = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10'];
+const avatarNumbers = {
+  MIN: 1,
+  MAX: 10,
+};
```

Comment:
молодец. Но есть не большое НО) 


```suggestion
const AvatarNumbers = {
  MIN: 1,
  MAX: 10,
};
```

Перечисления пишутся с большой буквы)